package.json

```diff
@@ -11,7 +11,8 @@
     "lint": "eslint src --ext .ts --max-warnings 0",
     "types": "tsc",
     "postversion": "git push && git push --tags",
-    "prepack": "npm run build && clean-package",
+    "prepare": "npm run build",
```

Comment:
This is unnecessary as the workflow already handles building.

```suggestion
```

Reply:
woops, removed 👍 Thx